downstream/modules/platform/proc-using-postinstall.adoc

```diff
@@ -29,6 +29,22 @@ controller_license_file=/full_path_to/manifest_file.zip
 controller_postinstall_repo_url=https://your_cac_scm_repo
 controller_postinstall_dir=/full_path_to_where_you_want_the pulled_content_to_reside
 ----
++
+
+. The controller_postinstall_repo_url variable can be used to define the postinstall repository URL which may include authentication information.
+
++
+----
+http(s)://<host>/<repo>.git (public repository without http(s) authentication)
+http(s)://<user>:<password>@<host>:<repo>.git (private repository with http(s) authentication)
+git@<host>:<repo>.git (public/private repository with ssh authentication)
+----
++
+
+[NOTE]
+====
+When using ssh based authentication, the installer does not configure anything for you, so you will need to configure everything on the installer node.
```

Comment:
Suggest changing "will need to" to "must". IBM Style: Use "must", not "have to" or "need to"